cmd/flags_headers.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/multiformats/go-multiaddr"
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/celestiaorg/celestia-node/node"
+)
+
+var (
+	headersTrustedHashF = "headers.trusted-hash"
+	headersTrustedPeerF = "headers.trusted-peer"
+)
+
+func HeadersFlags() *flag.FlagSet {
+	flags := &flag.FlagSet{}
+
+	flags.String(
+		headersTrustedHashF,
+		"",
+		"Hex encoded block hash. Starting point for header synchronization",
```

Comment:
```suggestion
		"Hex-encoded block hash checkpoint for header synchronization",
```

Reply:
Checkpoint actually sounds as if this hash is checked along the way and you start syncing with earlier hashes. Is that the case now? If not, starting point is more accurate, or simply:
```suggestion
		"Hex encoded block hash. Used to subjectively intitialize header synchronization",
```

Reply:
Go with Ismail's suggestion!

Reply:
I agree though it requires the reader to be aware of the subjective initialization concept. Starting check/point is more readable imo.

Reply:
I also changed block => header in suggestion.

Reply:
If you still think "subjectively initialize" is more readable I apply suggestion.

Reply:
I think subjectively initialize is good.

Reply:
mkay